thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/entity/DatasetConfigIndex.java

```diff
@@ -3,6 +3,7 @@
 public class DatasetConfigIndex extends AbstractIndexEntity {
   String dataset;
   boolean active;
+  boolean requiresCompletenessCheck;
```

Comment:
it's stronger than just enabling completeness check -- the detector will require a check to be performed before applying anomaly detection